flight/uavobjectlib/Makefile

```diff
@@ -66,8 +66,15 @@ LDFLAGS += -Wl,--fatal-warnings
 SRC += $(filter-out $(OPUAVSYNTHDIR)/uavobjectsinit.c,$(wildcard $(OPUAVSYNTHDIR)/*.c))
 NO_AUTO_UAVO = YES
 
+# Backup the environmental variable before entering firmware-common.mk
```

Comment:
Why do you need to do this?  The uavobjects don't need to know what RTOS it is-- why does it need to get "put back"?